routes/products.js

```diff
@@ -23,4 +41,55 @@ router.get('/:id', (req, res) => {
 		});
 });
 
+
+router.get('/', (req, res) => {
+	var productType = req.query.product;
+
+	Product.findOne({product: productType})
+		.then((doc) => {
+			if(!doc) {
+				const SPARQL = `
+        SELECT ?manufacturerLabel
+        WHERE
+        {
+          ?product ?label \"` + productType + `\"@en.
+          ?manufacturer wdt:P1056 ?product.
+          SERVICE wikibase:label {bd:serviceParam wikibase:language "en" }
+        }`
+				const url = wdk.sparqlQuery(SPARQL);
+
+				rp(url)
+					.then((body) => {
+						var manufacturers = [];
```

Comment:
These queries should be stored to the MongoDB (I know that pr still hasn't been approved) and then the results should be retrieved from there. The MongoDB should have the following:

It should contain one document of all manufacturers with ESG scores. These will allow us to to append ESG scores to everything we retrieve. If a manufacturer is not found in that list, we should append a random ESG score and add the manufacturer to that list (for now). 

The manufacturers should then be sorted according to this list when they are being retrieved.

Reply:
See this for adding to the Mongo:
https://github.com/reisman-internship-2019-greenmap/GreenMap-Database/blob/%238-esg-and-product-types/SPARQL%20MongoDB%20Fill%20Example.js

Reply:
See this for querying the mongo:
https://github.com/reisman-internship-2019-greenmap/GreenMap-Database/blob/%238-esg-and-product-types/Mongo%20Database%20Access%20Example.js